lib/commands/getAppConfig.js

```diff
@@ -0,0 +1,28 @@
+const wsContentTypes = require('../api/wsContentTypes');
+const chainPromise = require('../utils/chainPromise');
+const {getAppConfigRequest, getAppConfigResponse, getAppConfigError} = require('../texts');
+
+async function getAppConfig({appContext, authContext, webSockets, logger}) {
+	logger.log(getAppConfigRequest(appContext._context.configId));
```

Comment:
Not sure that we should do any logging for this function - this is just utility function, that do not affect test execution.